usecases/usecase/usecase.go

```diff
@@ -114,6 +117,10 @@ func (u *UseCaseBase) IsCompatibleEntityType(entity spineapi.EntityRemoteInterfa
 		return false
 	}
 
+	if u.allEntityTypesValid {
```

Comment:
Please add a test for this scenario, e..g by adding the following in usecase_test.go link 26:

```golang
	s.uc.allEntityTypesValid = true
	result = s.uc.IsCompatibleEntityType(payload.Entity)
	assert.True(s.T(), result)
```